### PR TITLE
Improve docblocks for `tribe_get_event_website_link` function and fil…

### PR DESCRIPTION
**Pull request by Sjaure**
…ters

### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/functions/template-tags/link.php [hunk 1]
```diff
@@ -381,13 +381,18 @@ function tribe_get_event_link( $post_id = null, $full_link = false ) {
...
+ *
+ * This function returns the URL associated with the event and outputs it as an HTML anchor (`<a>`) element.
+ * It also allows customization of the link label and target through filters.
+ *
+ * @since 3.0
  *
- * @param null|object|int $event The event object or ID.
- * @param ?string         $label The link label.
- * @param string          $target The link target.
+ * @param null|object|int $event  The event object or ID. Defaults to null.
```
Comment (Camwyn) [inline src/functions/template-tags/link.php:391]:
```suggestion
 * @param null|object|int $event  The event object or ID. Defaults to null - which will use global `$post`.
```

File: src/functions/template-tags/link.php [hunk 2]
```diff
@@ -381,13 +381,18 @@ function tribe_get_event_link( $post_id = null, $full_link = false ) {
...
+ * @param null|string     $label  Optional. The text label for the link. Defaults to the event URL if not provided.
+ * @param string          $target Optional. The target attribute for the link. Defaults to '_self'.
```
Comment (Camwyn) [inline src/functions/template-tags/link.php:393]:
```suggestion
 * @param string          $target Optional. The target attribute for the link. Defaults to '_self'. Allowed values are: '_self', '_blank', '_parent', '_top', '_unfencedTop'.
```

**Review by codingmusician — DISMISSED** [review]

**Review by Camwyn — CHANGES_REQUESTED** [review]
I think these two need a bit more info.

File: src/functions/template-tags/link.php (hunk 1 above)
Comment (Sjaure) [reply src/functions/template-tags/link.php]: Done @Camwyn

File: src/functions/template-tags/link.php (hunk 2 above)
Comment (Sjaure) [reply src/functions/template-tags/link.php]: Done

**Review by Camwyn — APPROVED** [review]

**Pull request merged**